README.md

```diff
@@ -7,3 +7,27 @@ Initially the first step in this project is to have a Rust ecosystem developed t
 Now that we have some code developed to control the Teensy, some prototyping is necessary to interface sensors and motors to the Teensy. The IMU chip for navigation we're using is the MPU-9250. This has well developed documentation and a nice dev board available from [Sparkfun](https://www.sparkfun.com/products/13762) A demo in Rust is implemented to access the IMU readings from the Teensy. The accelerometer, rate gyro, and magnetometer on the chip all have different bandwidths, so they must all be queried separately. The accelerometer can be queried at 4 kHz, the rate gyro can be queried at 8000 Hz, and the magnetometer can be queried at 8 Hz. The [demo program](https://github.com/e-belski/rust-copter/tree/master/demos/mpu9250-getting-started) shows how to sample the accelerometer on the MPU chip.
 
 More documentation to come as we develop it.
+
+## Getting started
+
+[Install `git-lfs`](https://git-lfs.github.com). We use `git-lfs` to manage larger files, like PDF reference manuals and 3D models, in this repository.
```

Comment:
Doesn't need to be all 3D models. STL and STEPs are text pretty much text. We'll just ignore them in the diffs.